jbs/src/main/java/org/openjdk/skara/jbs/Backports.java

```diff
@@ -73,8 +73,8 @@ private static boolean isNonScratchVersion(String version) {
             log.warning("Issue " + issue.id() + " has multiple valid fixVersions - ignoring");
             return Optional.empty();
         }
-        if (issue.properties().containsKey("customfield_10006") && issue.properties().get("customfield_10006").isObject()) {
-            return JdkVersion.parse(versionString.get(0), issue.properties().get("customfield_10006").get("value").asString());
+        if (issue.properties().containsKey("customfield_10006")) {
+            return JdkVersion.parse(versionString.get(0), issue.properties().get("customfield_10006").asString());
```

Comment:
Would you mind creating a variable naming the custom field? For example:

```java
var resolvedInBuild = "customfield_10006";
if (issue.properties().containsKey(resolvedInBuild)) {
```

Reply:
This exist in quite a few places already, and properly fixing it would require a larger refactoring. So I think we should do that separately.

Reply:
Ok, sounds good!